### PR TITLE
Google Classroom Share Button

### PR DESCRIPTION
**Pull request by cforkish**
This PR adds a [Google Classroom Share Button](https://developers.google.com/classroom/guides/sharebutton) to the unit overview page when the section login type is `google_classroom` and the current user has the `google_oauth2` provider type.

**Before:**
Screen Shot 2020-08-28 at 4 08 41 PM

**After:**
Screen Shot 2020-08-28 at 4 08 12 PM

This is a draft PR because the feature still needs to be reviewed by Amanda and Marina, but I want to go ahead and get the conversation started in the meantime.

### Future work
- [LP-1521] will add instrumentation for this feature.
- We plan to discuss additional places we might want to add this button, e.g. for assigning whole units.

## Links
- [LP-1522]

## Testing story
- added new unit test to verify conditional rendering of button
- added new storybook stories displaying the button

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

[LP-1521]: https://codedotorg.atlassian.net/browse/LP-1521
[LP-1522]: https://codedotorg.atlassian.net/browse/LP-1522

### CODE REVIEW

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/GoogleClassroomShareButton.jsx]: NIT: Looks like you don't need an arrow function here. Should just be able to do `gapiReady() { ... }`

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/GoogleClassroomShareButton.jsx]: Does this need to be an id, or can it be a ref or class? (or can it be removed entirely?) In general, it's best to [avoid ids in React](https://www.javascriptstuff.com/use-refs-not-ids/) where possible.

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/GoogleClassroomShareButton.jsx]: Will we need to add apis.google.com to our list of "Sites to Unblock" in the [technical requirements](https://support.code.org/hc/en-us/articles/202591743-Technical-requirements-for-Code-org) and [IT](https://code.org/educate/it) page?

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/GoogleClassroomShareButton.jsx]: If apis.google.com is down or unreachable, will this keep spinning forever? If so, should we add a timeout?

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/GoogleClassroomShareButton.jsx:128]: In case anybody else feels like they need some docs on this, here they are! https://developers.google.com/classroom/guides/sharebutton

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/GoogleClassroomShareButton.jsx]: NIT: I'd suggest using a somewhat more unique id. Given that this is abbreviated, I could see someone in the future adding an element with the same name and causing collisions.

File: apps/src/templates/progress/ProgressLesson.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/ProgressLesson.jsx]: FYI: shareUrl generally refers to the unique url of a user's project. i.e. 
https://studio.code.org/projects/applab/pM1f-UJm2YPi_MqHIpTMbaKgFrJGgifalAd1eaJrIhU
would be a shareUrl (accessed by `dashboard.project.getShareUrl()`). I'd recommend renaming this to something like `levelUrl` to clear up confusion.

File: apps/src/templates/teacherDashboard/shapes.jsx
Comment (jmkulwik) [inline apps/src/templates/teacherDashboard/shapes.jsx:127]: Thanks for doing this cleanup!

File: apps/test/unit/templates/progress/ProgressLessonTeacherInfoTest.js
Comment (jmkulwik) [inline apps/test/unit/templates/progress/ProgressLessonTeacherInfoTest.js:4]: While you're here, mind replacing deprecatedChai with reconfiguredChai?

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: thanks for calling this out. i thought arrow functions were preferred for single-statement functions. [this article](https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1) set me straight on that!

i also discovered that arrow functions will auto-bind `this`, and thought that was nifty and started using it extensively in my follow-up metrics PR. but when i double-checked our style guide to see what is the preferred method of binding, i discovered that using arrow functions is considered ["very bad"](https://github.com/airbnb/javascript/tree/master/react#methods)!

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: this does need to be an id because that's what the google api uses to find the element in which to render the button. however, i think the concern raised in that article you linked doesn't really apply here since the id is passed in, rather than defined in the component itself. so the risk of duplicate ids isn't zero, but it's minimal.

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: nice catch! i didn't know about those pages. do you know who manages them?

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: makes sense. will update.

File: apps/src/templates/progress/ProgressLesson.jsx
Comment (cforkish) [reply apps/src/templates/progress/ProgressLesson.jsx]: 👍

File: apps/src/templates/teacherDashboard/shapes.jsx
Comment (cforkish) [reply apps/src/templates/teacherDashboard/shapes.jsx:127]: 👍

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: 👍

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: you thinking like set the start time on initial load then stop trying after some given interval? it's a pretty cheap operation so that could be overkill, but i'm happy to add it. 

another option would be to use exponential backoff (e.g. multiply the timeout interval by 2 each time) -- that might be a more robust than setting a specific "stop trying" interval in the case that it's just taking a really long time to load.

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: That's Josh Schulte and Marina. You're also welcome to update them yourself anytime and just make sure you get their signoff. I'm actually not sure if we need this one on the allow-list. I don't see other similar host names (but that could just be an accident). Josh and Marina would know. 

To update: The IT page lives here: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/educate/it.md
The technical requirements page is a blog post and instructions for updating are here and the following few threads: https://codedotorg.slack.com/archives/C039X1ZLX/p1592952987042400

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (clareconstantine) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: Thanks for sharing the link to the style guide - I hadn't seen (or didn't remember) this particular guidance!

File: apps/test/unit/templates/progress/ProgressLessonTeacherInfoTest.js
Comment (cforkish) [reply apps/test/unit/templates/progress/ProgressLessonTeacherInfoTest.js:4]: 👍

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: Hmm. I don't know if I have a specific preference. Just thinking it might be nice to communicate the state to users. Definitely not a hard requirement.

**Review by jmkulwik — APPROVED** [review]
Looks great!

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: @clareconstantine what do you think?

at the very least, it might be good to not render this component at all until the api is ready. right now we render the the label immediately without leaving space for the button, so when the api loads the label gets moved over, which looks a little wonky.

another option would be to render a loading spinner as a placeholder for the button, which we could replace with a "failed to load" icon on timeout if we take that approach. or it could just keep spinning indefinitely if we go with exponential backoff.

Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: per a [conversation with Marina](https://codedotorg.slack.com/archives/CA3KCSGTD/p1600112325161500) in slack, i'm going to not render anything until the api loads and add a timeout. i will also add metrics for api load time and failed loads in #36728

Comment (clareconstantine) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: I would lean towards not showing anything until the api loads - if we get a zendesk ticket of someone not seeing the button, they will still have an error in the console, and it just seems cleanest. Not sure how helpful this would be since most users might not know to check the console, but we could put a link to the sites to whitelist in a comment in the console if the api doesn't load, in case they check there.

**Review by clareconstantine — APPROVED** [review]

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (clareconstantine) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: oh just saw your comment :) sounds good!

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: turns out it doesn't actually need to be an id! the docs said "Specify either the ID of the container (string) or the DOM element itself", and i thought the latter option meant defining the element inline, which i couldn't figure out how to do. but it turns out it just meant a reference to the dom element, and now that i'm storing one of those anyway (in my followup PR) i will update this in my forthcoming updated metrics PR

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/GoogleClassroomShareButton.jsx]: Nit: Define these in `static defaultProps` like here: https://github.com/code-dot-org/code-dot-org/blob/90d53f9ae0436d55a119b17fabed984b6ccd574a/apps/src/templates/SchoolTypeDropdown.jsx#L51

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/GoogleClassroomShareButton.jsx]: NIT: This isn't in our style guide, but I always see us do `export default` at the class declaration. The exception to this is when we are using redux or radium. 
My two cents: In general, I've found this to be a useful convention so I know whether the component is using redux/radium and so I have to explicitly make the decision to include multiple components in a file.

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: will do. i seem to be really bad at picking canonical examples when i'm looking for conventions to copy! ;-)

File: apps/src/templates/progress/GoogleClassroomShareButton.jsx
Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: same as above... we don't seem to be very consistent about this and a lot of the components i've worked with are doing `export default connect` at the bottom of the file, but for non-redux components the convention you suggest makes sense.

Comment (jmkulwik) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: I should say that for this one, I think this might just be my personal preference (so not asking you to change it if you don't want to). If you have thoughts either way, I'd love to hear them!

Comment (cforkish) [reply apps/src/templates/progress/GoogleClassroomShareButton.jsx]: for things like this i'm not very opinionated, other than the opinion that i like consistency. i put your preference in the style guide proposal and we can see if anyone has a stronger preference in the other direction!

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [inline apps/src/templates/progress/googlePlatformApiRedux.js:33]: While we _can_ trigger ajax requests in redux, I would recommend against it. Redux is a global, predictable, state container. When it starts doing too much, it becomes difficult to reason about. It should only handle mutations in state. When an engineer opens a redux file, they should expect to see how various input map to various outputs. If we start having redux trigger ajax requests and multiple dispatches, redux stops being a state container, starts to turn into a global "does everything" manager.

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [inline apps/src/templates/progress/googlePlatformApiRedux.js:76]: This one is a bit confusing to me. If I finished loading the api, but the api errored, should `loaded` be true or false? Here it seems we are overloading `loaded` to mean "successfully loaded"

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [inline apps/src/templates/progress/googlePlatformApiRedux.js:21]: loadStartTime looks like it's acting as a local variable. It shouldn't be in the global redux store.

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (cforkish) [reply apps/src/templates/progress/googlePlatformApiRedux.js:33]: after all the reading i've done on async actions in redux, i have to disagree here. or rather, argue that this is just an async mutation of state.

first, my interpretation is that the redux file isn't about mapping various inputs to various outputs, but rather, mapping various _actions_ to various _states_ – which i think is exactly what is happening here.

the official redux docs have a whole [section on api fetching](https://redux.js.org/tutorials/essentials/part-5-async-logic), which implies they consider an async mutation of state that uses remotely-fetched data to update the local state to be a valid use of the redux store. in this case we're dispatching an action to make a request to load the api, which results in a mutation of state to either `loaded=true` or `failed=true` (with the state of `loading=true` in the interim).

all that said, what are you suggesting i should do here instead? just move all the api loading stuff into a different file? or some more fundamental change to the logic?

File: apps/src/templates/progress/ProgressLessonTeacherInfo.jsx
Comment (clareconstantine) [inline apps/src/templates/progress/ProgressLessonTeacherInfo.jsx]: do we need to check for the section's existence here before getting it's id?

File: apps/src/templates/progress/ProgressLessonTeacherInfo.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/ProgressLessonTeacherInfo.jsx]: Nit (trying to remember) we decided to bind these in the constructor, right?

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (cforkish) [reply apps/src/templates/progress/googlePlatformApiRedux.js:76]: yes, that's what it means -- i.e. "the api is loaded". i initially had another state property for `failed`, but a) it was never used, and b) setting `loaded = action.success; failed = !action.success` on consecutive lines seemed redundant, so i removed it. but i can add it back if you think it would make this clearer. or i can rename it to `successfullyLoaded`, but i hoped that was implied by `loaded = action.success`. but since it made you pause, i should probably rename it?

if our redux store persisted between route changes i think there would be more value in keeping track of failed loads in case we want to stop trying or something like that. but alas this state is super local and transient, so i decided to err on the side of simplicity.

File: apps/src/templates/progress/ProgressLessonTeacherInfo.jsx
Comment (jmkulwik) [inline apps/src/templates/progress/ProgressLessonTeacherInfo.jsx]: NIT: rather than adding this style inline, consider adding another style to the styles object with just marginBottom: 5 and using spread to combine the two.

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (cforkish) [reply apps/src/templates/progress/googlePlatformApiRedux.js:21]: can you explain why it looks like it's acting as a local variable? it's bound to the state, which is keeping track of the state of the api loading process, including when it starts.

File: apps/src/templates/progress/ProgressLessonTeacherInfo.jsx
Comment (cforkish) [reply apps/src/templates/progress/ProgressLessonTeacherInfo.jsx]: hah! we most certainly did! i'll also update `onClickHiddenToggle` while i'm at it. ;-)

File: apps/src/templates/progress/ProgressLessonTeacherInfo.jsx
Comment (cforkish) [reply apps/src/templates/progress/ProgressLessonTeacherInfo.jsx]: 👍

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [reply apps/src/templates/progress/googlePlatformApiRedux.js:33]: (Commenting in reverse order) 
My suggestion would be to use this redux file for only the state portions and move the api loading into a "loader" module. The "loader" module would then be what dispatches the actions.

Comment (jmkulwik) [reply apps/src/templates/progress/googlePlatformApiRedux.js:33]: Redux does offer support for api fetching, but I have yet to see it used elegantly. It almost by definition breaks "mapping various actions to various states" - an action is triggered that starts a load process, but it could end in 1 of 3 different states (loading, success, and error)

Comment (jmkulwik) [reply apps/src/templates/progress/googlePlatformApiRedux.js:33]: If you're up for it, I'd prefer to raise this question to the whole engineering org and allow multiple people to weigh in.

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [reply apps/src/templates/progress/googlePlatformApiRedux.js:21]: Looks like that is true for `loading` as well (not sure if I'm missing something)

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [reply apps/src/templates/progress/googlePlatformApiRedux.js:76]: I was originally going to suggest two action types: "LOADING_GAPI_SUCCES" and "LOADING_GAPI_FAIL" or something similar.

Comment (jmkulwik) [reply apps/src/templates/progress/googlePlatformApiRedux.js:76]: But I'm downgrading this comment to a NIT :)

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [inline apps/src/templates/progress/googlePlatformApiRedux.js:65]: Question: Why not use `googlePlatformApi.loaded` directly in the connect portion of the react component?

File: apps/test/unit/templates/progress/googlePlatformApiReduxTest.js
Comment (jmkulwik) [inline apps/test/unit/templates/progress/googlePlatformApiReduxTest.js:14]: 🎉

**Review by clareconstantine — COMMENTED** [review]
I agree with Jessie's thoughts on moving the actual ajax requests out of redux, but otherwise this looks good to me! And thanks for figuring out those tests!

[edit: I hadn't refreshed the page and see your discussion! I will go take a look at those docs, and am approving the changes - we can follow up with a refactor if we decide as a team not to use this pattern.]

File: pegasus/sites.v3/code.org/public/educate/it.md
Comment (jmkulwik) [inline pegasus/sites.v3/code.org/public/educate/it.md:33]: 🎉

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [reply apps/src/templates/progress/googlePlatformApiRedux.js:21]: (oops missed your comment until just now)
By "local variable" I meant that I don't think I see it being used anywhere outside `googlePlatformApiRedux` yet since it's in the redux store, I could be accessed from anywhere in the application.

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (cforkish) [reply apps/src/templates/progress/googlePlatformApiRedux.js:76]: haha, one of the guides i read specifically called out that whether to separate those into separate actions or include a bool value is a matter of preference. since you've downgraded the importance of this i'm going to leave it as is. :)

File: apps/src/templates/progress/ProgressLessonTeacherInfo.jsx
Comment (cforkish) [reply apps/src/templates/progress/ProgressLessonTeacherInfo.jsx]: nope!
```
    const sectionId = (section && section.id.toString()) || '';
    const showHiddenForSectionToggle =
      section && scriptAllowsHiddenStages && !hasNoSections;
```
that said, i'm updating that file anyway so i'll add it for clarity

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (cforkish) [reply apps/src/templates/progress/googlePlatformApiRedux.js:65]: we only register the reducer if we meet the section and user requirements for showing the button, so when that's not the case `!!state.googlePlatformApi === false`, and i don't want consumer components to need to know that detail.

also just as a stylistic preference -- the components that consume this don't really care about whether or not some google platform is loaded, they just care about whether they can render the google button.

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (cforkish) [reply apps/src/templates/progress/googlePlatformApiRedux.js:21]: i'm confused about why that's an issue. redux doesn't support the concept of internal properties so these are implicitly exposed in the redux tree, but that doesn't change the fact that they _are_ necessary properties of the state of the application that the loading process needs to be able to query.

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (cforkish) [reply apps/src/templates/progress/googlePlatformApiRedux.js:33]: re. first comment:
if this were a long complicated file i would totally agree, and i definitely did consider doing that initially, but in this case i would argue that separating the loading logic out would actually make all this harder to read because you'd need both files open side-by-side to make sense of it, and it would likely add a lot more lines of boilerplate code. so in this case where the file is less than 150 lines, i personally think it's easiest to understand having it all in one place.

another thing i like about keeping it in one file is that all of the state and logic is internal to this loading process, so right now we only export the reducer, `loadGooglePlatformApi`, and `canShowGoogleShareButton` (plus `GOOGLE_PLATFORM_API_ID` for testing), which are the only parts a consumer should care about. the only part of the state that can be subscribed to (without looking in this file or examining the state at runtime) is the `loaded` property, and even that is obscured by `canShowGoogleShareButton`. so to the extent that someone might view the list of exports as the "external api", i think this module is exposing exactly what it should. if we separate it out into a loader, everything would be exposed, giving it the potential to be misused.

additionaly, it turns out the redux docs actually [encourage](https://redux.js.org/style-guide/style-guide#structure-files-as-feature-folders-or-ducks) putting all the logic in a single file if you're not using the "feature folder" approach (which we most definitely are not). not to mention that in the context of our codebase i think this is the general expectation.

Comment (cforkish) [reply apps/src/templates/progress/googlePlatformApiRedux.js:33]: re. second comment:
what is the alternative in an app that uses redux and requires async requests? _something_ has to track the progress of the request, and synchronizing some external object with the redux state doesn't sound elegant to me.

we know that there are three actions that need to update the state of this redux branch:

1) page load => start loading api :: state.loading = true
2) load fails => finish loading api (false) :: state.loading = false, state.loaded = false
3) load succeeds => finish loading api (success=true) :: state.loading = false, state.loaded = true

so we actually have three actions resulting in three different states (though in this case the failure case isn't made explicit, because our application doesn't need to know about it).

it is true that a single action initiates the transition from the initial state to one of three resulting states, but that is the only valid way to initiate that transition (there's no valid way to set `loaded=true` other than dispatching that first async action). and it does require a subsequent event (the api finishes loading or fails) to transition past the state caused by the first action.

also, i guess i'm not sure what this comment is suggesting i should change. even if i move the loading logic to a different module, it would still need to dispatch an async action, because _by definition_ the transition to `loading=true` needs to be atomically bound to the initiation of the loading process. do you have some other proposal for how our redux-based application should handle async requests?

Comment (cforkish) [reply apps/src/templates/progress/googlePlatformApiRedux.js:33]: re. comment 3:
i personally don't think this is the time/place to initiate that conversation with the larger team unless we're at an impasse here. this feature is _way_ overdo and that's a big conversation to have as an org, so in my opinion the fact that we already do this a fair amount in our code means we're not introducing some new bad practice, just deferring the cleanup of an (arguably) existing one.

i think your first comment mostly amounts to an issue of style and code organization, and i don't have a strong enough opinion about it to delay this feature anymore, so if you feel strongly that this stuff should be separated into two files i will do that.

as for your second comment, i would suggest that you come up with a proposal for how we _should_ keep redux store in sync with the state of ajax requests before raising the issue with the broader team.

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [reply apps/src/templates/progress/googlePlatformApiRedux.js:65]: 👍 - Thank you!

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [reply apps/src/templates/progress/googlePlatformApiRedux.js:33]: Thank you for the context on timing/urgency for this feature and for the very well thought out comments. You are completely right - it's definitely _not_ worth delaying this feature to get this all figured out. I'm going to go ahead and retract this suggestion on this PR. This PR was a great catalyst to get this discussion started, but the discussion and decision should be separate from the PR.

File: apps/src/templates/progress/googlePlatformApiRedux.js
Comment (jmkulwik) [reply apps/src/templates/progress/googlePlatformApiRedux.js:21]: Same as my above redux comment (changing to non-blocking)

**Comment by jmkulwik** [comment]
I'm already marked as signed off, so here's my second sign off :) ✔️

**Pull request merged**
